templates/packages/order_edit.html

```diff
@@ -0,0 +1,254 @@
+                            <div class="col-auto">
+                                <button type="button" class="btn btn-outline-dark" data-bs-dismiss="modal" aria-label="Close">בטל</button>
+                            </div>
+                            <div class="col">
+
+                            </div>
+                            <div class="col-auto">
+                                <button type="submit" class="btn btn-outline-success" onclick="submitPackageForm()">אשר</button>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+    <script>
+        //this is so pretty ^_^
+        document.getElementById('package-modal').addEventListener('show.bs.modal', function (event) {
+            var origin_street = event.relatedTarget.getAttribute('data-bs-origin-street')
+            var origin_street_number = event.relatedTarget.getAttribute('data-bs-origin-street-number')
+            var origin_city = event.relatedTarget.getAttribute('data-bs-origin-city')
+            var origin_area = event.relatedTarget.getAttribute('data-bs-origin-area')
+
+            document.getElementById('origin-street').value = origin_street
+            document.getElementById('origin-street-number').value = origin_street_number
+            document.getElementById('origin-city').value = origin_city
+            document.getElementById('origin-area').value = origin_area
+
+            var destination_street = event.relatedTarget.getAttribute('data-bs-destination-street')
+            var destination_street_number = event.relatedTarget.getAttribute('data-bs-destination-street-number')
+            var destination_city = event.relatedTarget.getAttribute('data-bs-destination-city')
+            var destination_area = event.relatedTarget.getAttribute('data-bs-destination-area')
+
+            document.getElementById('destination-street').value = destination_street
+            document.getElementById('destination-street-number').value = destination_street_number
+            document.getElementById('destination-city').value = destination_city
+            document.getElementById('destination-area').value = destination_area
+
+            var full_name = event.relatedTarget.getAttribute('data-bs-full-name')
+            var phone_number = event.relatedTarget.getAttribute('data-bs-phone-number')
+            var notes = event.relatedTarget.getAttribute('data-bs-notes')
+            var rate = event.relatedTarget.getAttribute('data-bs-rate')
+            document.getElementById('full-name').value = full_name
+            document.getElementById('phone-number').value = phone_number
+            document.getElementById('notes').value = notes
+            document.getElementById('rate').value = rate
+        })
+        function submitPackageForm() {
+            //add validation here
+            document.getElementById('package-form').submit()

```

Comment:
You have an error in the view caused by submitting.

![form](https://i.imgur.com/2eEoMaM.png)
![error](https://i.imgur.com/tyehgFW.png)

```

Traceback:

File "/home/tomtzook/projects/others/dandeduck/package-tracking-web/venv/lib/python3.8/site-packages/django/core/handlers/exception.py" in inner
  34.             response = get_response(request)

File "/home/tomtzook/projects/others/dandeduck/package-tracking-web/venv/lib/python3.8/site-packages/django/core/handlers/base.py" in _get_response
  115.                 response = self.process_exception_by_middleware(e, request)

File "/home/tomtzook/projects/others/dandeduck/package-tracking-web/venv/lib/python3.8/site-packages/django/core/handlers/base.py" in _get_response
  113.                 response = wrapped_callback(request, *callback_args, **callback_kwargs)

File "/home/tomtzook/projects/others/dandeduck/package-tracking-web/packages/views.py" in order_edit_view
  151.         save_changes_to_cookies(request, order, response)

File "/home/tomtzook/projects/others/dandeduck/package-tracking-web/packages/views.py" in save_changes_to_cookies
  219.     destination_address = get_or_create_destination_address(request)

File "/home/tomtzook/projects/others/dandeduck/package-tracking-web/packages/views.py" in get_or_create_destination_address
  247.     return get_or_create_address(destination_city_name, destination_area, destination_street, destination_street_number)

File "/home/tomtzook/projects/others/dandeduck/package-tracking-web/packages/views.py" in get_or_create_address
  261.     return Address.objects.get_or_create(city=destination_city, street_name=street_name, street_number=street_number)[0]

File "/home/tomtzook/projects/others/dandeduck/package-tracking-web/venv/lib/python3.8/site-packages/django/db/models/manager.py" in manager_method
  82.                 return getattr(self.get_queryset(), name)(*args, **kwargs)

File "/home/tomtzook/projects/others/dandeduck/package-tracking-web/venv/lib/python3.8/site-packages/django/db/models/query.py" in get_or_create
  538.             return self.get(**kwargs), False

File "/home/tomtzook/projects/others/dandeduck/package-tracking-web/venv/lib/python3.8/site-packages/django/db/models/query.py" in get
  399.         clone = self.filter(*args, **kwargs)

File "/home/tomtzook/projects/others/dandeduck/package-tracking-web/venv/lib/python3.8/site-packages/django/db/models/query.py" in filter
  892.         return self._filter_or_exclude(False, *args, **kwargs)

File "/home/tomtzook/projects/others/dandeduck/package-tracking-web/venv/lib/python3.8/site-packages/django/db/models/query.py" in _filter_or_exclude
  910.             clone.query.add_q(Q(*args, **kwargs))

File "/home/tomtzook/projects/others/dandeduck/package-tracking-web/venv/lib/python3.8/site-packages/django/db/models/sql/query.py" in add_q
  1290.         clause, _ = self._add_q(q_object, self.used_aliases)

File "/home/tomtzook/projects/others/dandeduck/package-tracking-web/venv/lib/python3.8/site-packages/django/db/models/sql/query.py" in _add_q
  1315.                 child_clause, needed_inner = self.build_filter(

File "/home/tomtzook/projects/others/dandeduck/package-tracking-web/venv/lib/python3.8/site-packages/django/db/models/sql/query.py" in build_filter
  1251.         condition = self.build_lookup(lookups, col, value)

File "/home/tomtzook/projects/others/dandeduck/package-tracking-web/venv/lib/python3.8/site-packages/django/db/models/sql/query.py" in build_lookup
  1116.         lookup = lookup_class(lhs, rhs)

File "/home/tomtzook/projects/others/dandeduck/package-tracking-web/venv/lib/python3.8/site-packages/django/db/models/lookups.py" in __init__
  20.         self.rhs = self.get_prep_lookup()

File "/home/tomtzook/projects/others/dandeduck/package-tracking-web/venv/lib/python3.8/site-packages/django/db/models/lookups.py" in get_prep_lookup
  70.             return self.lhs.output_field.get_prep_value(self.rhs)

File "/home/tomtzook/projects/others/dandeduck/package-tracking-web/venv/lib/python3.8/site-packages/django/db/models/fields/__init__.py" in get_prep_value
  1825.         return int(value)

Exception Type: ValueError at /partners/KSP/7b5b88bb-b138-4d55-a055-9c4bd2de3a2e/
Exception Value: invalid literal for int() with base 10: ''
```

Reply:
Yes, this is because there is no validation, you are basically submitting null for an int. I opened a separate issue #23 for validation since it will be fixed in a separate pr